experiments/ClimaEarth/cli_options.jl

```diff
@@ -2,107 +2,137 @@ import ArgParse
-        arg_type = Int
-        default = 400
-        "--dt_atmos"
-        help = " Atmos simulation time step"
-        arg_type = String
-        "--dt_land"
-        help = " Land simulation time step"
-        arg_type = String
-        "--dt_ocean"
-        help = " Ocean simulation time step"
+        ### ClimaCoupler flags
+        # Simulation-identifying information
+        "--config_file"
+        help = "A yaml file used to set the configuration of the coupled model [\"config/ci_configs/amip_default.yml\" (default)]"
         arg_type = String
-        "--dt_seaice"
-        help = " Sea Ice simulation time step"
+        default = "config/ci_configs/amip_default.yml"
+        "--job_id"
+        help = "A unique identifier for this run, defaults to the config file name"
         arg_type = String
-        "--energy_check"
-        help = "Boolean flag indicating whether to check energy conservation"
-        arg_type = Bool
-        default = false
-        "--ci_plots"
-        help = "Boolean flag indicating whether to make CI plots"
-        arg_type = Bool
-        default = false
-        "--conservation_softfail"
-        help = "Boolean flag indicating whether to soft fail on conservation errors"
+        default = nothing
+        "--print_config_dict"
+        help = "Boolean flag indicating whether to print the final configuration dictionary [`true` (default), `false`]"
         arg_type = Bool
-        default = false
+        default = true
         "--mode_name"
         help = "Mode of coupled simulation. [`amip` (default), `slabplanet`, `slabplanet_aqua`, `slabplanet_terra`, `slabplanet_eisenman`]"
         arg_type = String
         default = "amip"
-        "--mono_surface"
-        help = "Boolean flag indicating whether (1st order) monotone and conservative remapping is applied."
+        "--coupler_toml_file"
+        help = "An optional toml file used to overwrite the default model parameters."
+        default = nothing
+        # Computational simulation setup information
+        "--unique_seed"
+        help = "Boolean flag indicating whether to set the random number seed to a unique value [`false` (default), `true`]"
         arg_type = Bool
         default = false
-        "--turb_flux_partition"
-        help = "Method to partition turbulent fluxes. [`PartitionedStateFluxes`, `CombinedStateFluxesMOST` (default)]"
+        "--FLOAT_TYPE"
+        help = "Floating point precision  [`Float64` (default), `Float32`]"
         arg_type = String
-        default = "CombinedStateFluxesMOST"
+        default = "Float64"
+        "--device"
+        help = "Device type to use [\"auto\" (default), \"CPUSingleThreaded\", \"CPUMultiThreaded\", \"CUDADevice\"]"
+        arg_type = String
+        default = "auto"
+        # Time information
+        "--t_end"
+        help = "[REQUIRED] End time of the simulation (no default) [allowed formats: \"Nsecs\", \"Nmins\", \"Nhours\", \"Ndays\", \"Inf\"]"
```

Comment:
Maybe the default could just be t_end = t_start

Reply:
I'd rather make it 2 * default dt = 800secs, since we often run simulations for 2 steps to check interfaces

Reply:
Okay!